From: Carlos Maddela <e7appew@gmail.com>
Date: Thu, 5 Jan 2017 18:59:30 +1100
Subject: Fix display of manual.pdf.

Description: Fix display of manual.pdf.
 The PDF file is automatically compressed by Debhelper, so we need to
 account for this.
Author: Carlos Maddela <e7appew@gmail.com>
Forwarded: not-needed
Last-Update: 2017-01-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 show-manual.c    | 4 ++--
 welcome-window.c | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/show-manual.c b/show-manual.c
index c25342b..4abbe97 100644
--- a/show-manual.c
+++ b/show-manual.c
@@ -287,8 +287,8 @@ void ShowPDF(char *target)
 
    /* If no target is given, show the manual. */
 
-   if(!target) 
-   {    bi->path = g_strdup_printf("%s/manual.pdf",Closure->docDir); 
+   if(!target)
+   {    bi->path = g_strdup_printf("%s/manual.pdf.gz",Closure->docDir);
    }
    else 
       if(*target != '/') bi->path = g_strdup_printf("%s/%s",Closure->docDir, target); 
diff --git a/welcome-window.c b/welcome-window.c
index d30ebd9..63ac3df 100644
--- a/welcome-window.c
+++ b/welcome-window.c
@@ -139,8 +139,8 @@ void CreateWelcomePage(GtkNotebook *notebook)
    AboutText(box, _("\ndvdisaster creates error correction data to protect\n"
 		    "optical media (CD,DVD,BD) against data loss.\n"));
 
-   AboutTextWithLink(box, _("Please see the [manual] for typical uses of dvdisaster.\n\n"), 
-		     "manual.pdf");
+   AboutTextWithLink(box, _("Please see the [manual] for typical uses of dvdisaster.\n\n"),
+		     "manual.pdf.gz");
 
    AboutText(box, _("<i>New in this Version:</i>"));
 
